articles/mysql/flexible-server/concepts-workbooks.md

```diff
@@ -14,13 +14,13 @@ ms.topic: conceptual
 
 [!INCLUDE[applies-to-mysql-flexible-server](../includes/applies-to-mysql-flexible-server.md)]
 
-Azure Database for MySQL flexible server is now integrated with Azure Monitor workbooks. With workbooks, you get a flexible canvas for analyzing data and creating rich visual reports within the Azure portal. Workbooks allow you to tap into multiple data sources across Azure and combine them into unified interactive experiences. Workbook templates serve as curated reports that are designed for flexible reuse by multiple users and teams.
+Azure Database for MySQL Flexible Server is now integrated with Azure Monitor workbooks. With workbooks, you get a flexible canvas for analyzing data and creating rich visual reports within the Azure portal. Workbooks allow you to tap into multiple data sources across Azure and combine them into unified interactive experiences. Workbook templates serve as curated reports that are designed for flexible reuse by multiple users and teams.
 
 When you open a template, you create a transient workbook that's populated with the contents of the template. With this integration, the server links to workbooks and a few sample templates, which can help you monitor the service at scale. You can edit these templates, customize them to your requirements, and pin them to the dashboard to create a focused and organized view of Azure resources.
 
-In this article, you'll learn about the various workbook templates that are available for your flexible server.
+In this article, you'll learn about the various workbook templates that are available for your Flexible Server.
```

Comment:
```suggestion
In this article, you'll learn about the various workbook templates that are available for your flexible server.
```

Reply:
@ShannonLeavitt - Thanks for the suggestion.